pom.xml

```diff
@@ -27,7 +27,7 @@
 
         <azure-credentials.version>1.3</azure-credentials.version>
         <kubernetes-client.version>2.3.1</kubernetes-client.version>
-        <azure-commons.version>0.1.5</azure-commons.version>
+        <azure-commons.version>0.2.0-SNAPSHOT</azure-commons.version>
```

Comment:
Don't use snapshot version. Let's release commons first and then update this version.